scripts/examples/08-RPC-Library/36-Web-Servers/rtsp_video_server_wlan.py

```diff
@@ -30,7 +30,7 @@
 
 network_if = network.WLAN(network.STA_IF)
 network_if.active(True)
-network_if.connect('your-ssid', 'your-password')
+network_if.connect('your-ssid', 'your-password', timeout=30000)
```

Comment:
This arg may be removed in the future because it's not in upstream drivers, it's better to connect with something like this which works with all drivers:
```python
    wlan = network.WLAN(network.STA_IF)
    wlan.active(True)
    wlan.connect(WIFI_SSID, WIFI_PASS)
    while not wlan.isconnected():
        print("Trying to connect. Note this may take a while...")
        time.sleep_ms(500)
```

Reply:
Good idea.

Reply:
Done